### DISCUSSION TITLE
Never Leave Mattermost

### DISCUSSION THREAD

**Discussion by iomodo** [mattermost/community#60] [Practical Ideas]
Idea is to do "everything" in Mattermost without leaving it.
We could start with implementing an integration to console and to the DB.
Imagine slash commands like:
`/console ls`
`/console git commit -a -m "fun stuff"`
`/db select * from TABLE`

**Comment by esethna** [comment]
Yes! I think this might be related: The global command palette idea: https://github.com/mattermost/community/discussions/31

**Comment by chetanyakan** [comment]
Also receive emails in MM and messages from other chat applications using Matterbridge.
Plugins to integrate with Jira, Confluence, Github etc
